### ISSUE TITLE
Support for .mp3/.m4a/PSC chapters

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#895]
This feature was first requested in podverse-fdroid. Please see the description and screenshot in that ticket, but use this ticket going forward. https://github.com/podverse/podverse-fdroid/issues/47

**Comment by mitchdowney** [comment]
I think this feature is blocked until chapters can be extracted from ID3 tags in react-native-track-player. https://github.com/doublesymmetry/react-native-track-player/issues/1439

**Comment by Inrumpo** [comment]
> I think this feature is blocked until chapters can be extracted from ID3 tags in react-native-track-player. [doublesymmetry/react-native-track-player#1439](https://github.com/doublesymmetry/react-native-track-player/issues/1439)

This might be the case for .mp3 and .m4a chapters. Podlove Simple Chapters are stored in / written to the RSS feed. Podverse should be able to read them like any other tag.

**Comment by mitchdowney** [comment]
@Inrumpo good point. I'm going to create a separate issue for PSC chapters since they're fundamentally different than mp3/m4a chapters.

**Comment by mitchdowney** [comment]
@Inrumpo thanks for the update. We'd still like to add this, but it's blocked until #1179 is completed. After that is finished, we may have to make a PR into react-native-track-player to extract the chapter data.

**Comment by mitchdowney** [comment]
I think this is blocked due react-native-track-player not supporting the ability to grab any metadata tag we want.

https://github.com/doublesymmetry/react-native-track-player/issues/2008

**Comment by erifneerg** [comment]
From what it seems, the issue has been resovled @mitchdowney in issue [doublesymmetry/react-native-track-player# 2118](https://github.com/doublesymmetry/react-native-track-player/pull/2118). Hopeful this allows embedded chapters to be used in the app.

**Comment by mitchdowney** [comment]
@erifneerg wow thanks for the update! It appears according to the release notes that the chapters metadata is only available on iOS, not Android...I'd like to find out why Android isn't supported yet before embarking on feature support for this.

Also, I'm actually thinking seriously about beginning a full rewrite of the mobile app...and possibly the api...and possibly the whole Podverse tech stack 😳 it's over 5 years old at this point, and there are a TON of features in it I never imagined when beginning Podverse (mostly Podcasting 2.0 features).

If we go the full rewrite direction, then I would only be making bug fixes for the existing Podverse mobile app, and would save a new feature like mp3 embedded chapter support for the next version.

**Comment by mitchdowney** [comment]
We are archiving all Github issues and PRs that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
